.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -21,4 +21,4 @@ assignees: ''
  
  **Expected behavior:**
  
- **Minimal repro:**
```

Comment:
I am pretty sure that `repro` stands for `reproduction` (Or something along the lines of this). To make it clearer we could just use the full word: `Minimal Reproduction`.